packages-native/actions/src/camera/TakePictureAdvanced.ts

```diff
@@ -12,6 +12,14 @@ type PictureSource = "camera" | "imageLibrary" | "either";
 type PictureQuality = "original" | "low" | "medium" | "high" | "custom";
 
 /**
+ * Take a picture using the camera or import one from the image library on the device.
+ *
+ * The result is an ImageMetaData object. Most items are self-explanatory.
+ *
+ * The FileName value will be empty for pictures taken on an iOS device. To get the right extension you can get that from the URI:
```

Comment:
This is not true anymore as the filename was fixed

Reply:
I still need to look into your fix. I think the URI is no longer relevant because the file the URI points to has already deleted anyway. Do you agree to take out the URI altogether?

Reply:
I tried the latest release with your fix, works perfectly. So I think it is best to remove URI from the ImageMetaData as it no longer serves any purpose. I will do that soon and update this PR.

Reply:
I removed the URI attribute from ImageMetaData and TakePictureAdvanced. I updated the documentation as well.

Reply:
I noticed the Travis CI build failed. It does not seem to be an issue with my commit but with other widgets.

Reply:
> @diego-antonelli I did the rebase. When building for packages\jsActions\native-mobile-resources, where does the script expect the test project to be?

https://github.com/mendix/widgets-resources/blob/08f9a95a131382bf5aca9090039277c69ac37115/packages/jsActions/native-mobile-resources/scripts/copyActionsToTestProject.ts#L6